app/aqua-ames.hoon

```diff
@@ -0,0 +1,81 @@
+::  This needs a better SDN solution.  Every ship should have an IP
+::  address, and we should eventually test changing those IP
+::  addresses.
+::
+::  For now, we broadcast every packet to every ship and rely on them
+::  to drop them.
+::
+/-  aquarium
+=,  aquarium
+=>  $~  |%
```

Comment:
extraneous `$~` ... I don't understand why this doesn't break everything, but it should still be removed

Reply:
Ha, good catch.  It doesn't break anything because (1) $~ is a mold, so it's a core with the normal subject in its context and (2) Ford compiles files as a list of expressions implicitly tisgar'd together.